.github/workflows/lint.yml

```diff
@@ -28,3 +28,5 @@ jobs:
         node-version: '16'
     - run: npm install
     - run: npm run lint
+    - run: .ci/scripts/test_basic.sh
+    - run: .ci/scripts/test_types_babel_esm.sh
```

Comment:
These two scripts effectively do:

```
npm install
npm run lint
npm run test:deps

npm install
npm run test:types
npm run test:babel
npm run test:esm
```

The repeated `npm install`s are wasteful. As well, all those other `npm run test:*` targets are already being run in regular test matrix here: https://github.com/elastic/apm-agent-nodejs/blob/main/test/script/run_tests.sh#L40-L74

Can we remove these two lines and delete those two ".ci/scripts/" files?

Reply:
See 7dd45f74418382c3e3c3300edb704fc6ca4999b3